data/human/jobs.txt

```diff
@@ -3042,3 +3042,129 @@ mission "Bounty Hunting (Marauder X)"
 	on complete
 		payment 750000
 		dialog phrase "generic fleet bounty hunting payment dialog"
+
+
+
+mission "Hunted Warning"
+	invisible
+	landing
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		attributes "rim" "south" "north" "dirt belt" "core" "frontier"
+	to offer
+		"combat rating" > 8103
+		or
+			not "chosen sides"
+			has "main plot completed"
+	on offer
+		log `Got noticed at a spaceport bar as a "famous warrior." Was warned to watch for pirates looking to make a name for themselves or bounty hunters looking to claim a bounty.`
+		conversation
+			`You walk into a bar in the spaceport looking for something to eat when a man sitting at a table in the corner looks up and takes notice of you. "It's Captain <last>, the famous warrior! Barkeep, give <first> a drink on my tab."`
+			choice
+				`	"Glad to get some recognition. I work hard out here."`
+					goto recognition
+				`	"Famous warrior?"`
+			`	"Of course! Word travels fast in the galaxy, and word on the hyperspace lanes is that you're quite the fighter."`
+			
+			branch recognition
+				"reputation: Republic" >= 0
+				"reputation: Free Worlds" >= 0
+				"reputation: Syndicate" >= 0
+			
+			`	"It doesn't help that you've pissed off the law, either," someone says, not sounding too happy.`
+			label recognition
+			`	More of the patrons of the bar begin to notice that you've entered and begin chattering amongst themselves. "Quite the force to be reckoned with, I've heard," you overhear one of them mumble.`
+			`	"You might want to watch your back," another says to you. "Plenty of pirates looking to make a name for themselves taking out such a famous captain, and plenty of bounty hunters out for a bounty you might have on your head without even knowing about it."`
+			choice
+				`	"I'll be fine. My ship is strong enough to stand against anything."`
+				`	"Maybe I should go into hiding for a while."`
+					goto hide
+			`	"Oh, I'm sure it is. You wouldn't have gotten this notorious if you weren't able to hold your all."`
+				goto end
+			label hide
+			`	"You'd need to hide for more than just a while, but laying low in a small ship might help to stay off the radar of any bounty hunters."`
+			label end
+			`	After a few more minutes of mumbling, the bar returns to like it was before you entered. The food you ordered arrives and you dig in.`
+				decline
+
+mission "Hunted"
+	invisible
+	landing
+	deadline 30
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		attributes "rim" "south" "north" "dirt belt" "core" "frontier"
+	to offer
+		"combat rating" > 8103
+		"armament detterence" >= 2
+		random < 3
```

Comment:
Thoughts on increasing the chance of this spawning with the player's armament deterrence? Peter has also talked about adding a condition for the number of ships the player has owned and unparked. (This would be shorthand for `"ship: Interceptor" + "ship: Transport" + ...`.) Maybe using that, we could instead have this scale with the player's fleet size. Then, just as raid fleets balance against large freighter fleets, bounty hunters could balance against large fleets in general by becoming more common the bigger your fleet is.

Reply:
One alternative is to use the result of the mission itself to increase its likelihood of offering, up to a certain amount. This is a little tricky since a failed mission is one in which you evaded the pursuers for 30 days or killed one of the "save" types then landed, and a completed mission is one in which you either killed all the strongest foes or had the "save" types, and landed on the destination planet.
(Given the low chance of landing on the correct destination planet at random after completing the mission, unless it is part of the player's usual trade route, it may be difficult to actually clear the more difficult mission versions.)

Another path is as the conversation hints, to use a negative reputation with governments to increase the chance of offering. Perhaps if 2 of the 3 hate you, then the rate increases. (This way could be too punishing for pirate playthroughs, but does make sense.)

I don't think we should allow the offer rate to increase above e.g. 10% as very capable players will simply disable and capture their pursuers.

Reply:
I don't want to use your reputation with governments to influence this mission, as you already have it hard enough by not being able to travel to a large swathe of space without being attacked.

>(Given the low chance of landing on the correct destination planet at random after completing the mission, unless it is part of the player's usual trade route, it may be difficult to actually clear the more difficult mission versions.)

Seems to me like #2131 would need to be a thing.